clients/assessment_component/src/assessment/pages/AssessmentPage/components/AssessmentForm/components/DeleteAssessmentDialog.tsx

```diff
@@ -0,0 +1,55 @@
+import { Trash2 } from 'lucide-react'
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  Button,
+} from '@tumaet/prompt-ui-components'
+
+interface DeleteAssessmentDialogProps {
+  open: boolean
+  onOpenChange: (open: boolean) => void
+  onConfirm: () => void
+  isDeleting?: boolean
+}
+
+export const DeleteAssessmentDialog = ({
+  open,
+  onOpenChange,
+  onConfirm,
+  isDeleting = false,
+}: DeleteAssessmentDialogProps) => {
+  const handleCancel = () => {
+    onOpenChange(false)
+  }
+
+  const handleConfirm = () => {
+    onConfirm()
+  }
+
+  return (
+    <Dialog open={open} onOpenChange={onOpenChange}>
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Reset Assessment</DialogTitle>
+          <DialogDescription>
+            Are you sure you want to reset this assessment? This action cannot be undone and will
+            reset this assessment and its comment permanently.
```

Comment:
Maybe a better word choice, since "resets it permanently" kind of implies that you can't redo it.

```suggestion
            Are you sure you want to reset this assessment? This action cannot be undone and you will have to complete this assessment and its comment again afterward.
```

Reply:
I like the suggestion